docs/guides/adding-custom-css.rst

```diff
@@ -0,0 +1,18 @@
+Adding Custom CSS
```

Comment:
Should make this more google friendly ("Adding custom css to your Sphinx project") or similar.